CHANGELOG.md

```diff
@@ -4,6 +4,42 @@ All notable changes to Zebra are documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-beta.3](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-beta.3) - 2021-12-17
```

Comment:
we should change this to the date when we tag the release

Reply:
I bumped the date in case we want to release today